app/views/providers/allocations/request_status/_repeat.html.erb

```diff
@@ -0,0 +1,60 @@
+<% if @allocations_view.repeat_allocation_statuses.present? %>
+
+  <table class="govuk-table ucas-info-table govuk-!-margin-bottom-8" data-qa="repeat-allocations-table">
+  <thead class="govuk-table__head">
+  <tr class="govuk-table__row">
+    <th class="govuk-table__header subject-col">
+      Organisation
+    </th>
+    <th class="govuk-table__header status-col">
+      Status
+    </th>
+    <th class="govuk-table__header status-col">
+      <span class="govuk-visually-hidden">Actions</span>
+    </th>
+  </tr>
+  </thead>
+  <tbody class="govuk-table__body">
+  <% @allocations_view.repeat_allocation_statuses.each do |allocation| %>
+    <tr class="govuk-table__row">
+      <th class="govuk-table__header" data-qa="provider-name">
+        <%= allocation[:training_provider_name] %>
+      </th>
+      <td class="govuk-table__cell">
+            <span class="govuk-tag govuk-tag--<%= allocation[:status_colour] %>">
+              <%= allocation[:status] %>
+            </span>
+      </td>
+      <td class="govuk-table__cell">
+        <% if allocation[:status] == AllocationsView::Status::YET_TO_REQUEST %>
+          <%= link_to "Confirm choice",
+                      new_provider_recruitment_cycle_allocation_path(
+                        @provider.provider_code,
+                        @provider.recruitment_cycle_year,
+                        allocation[:training_provider_code]),
+                      class: "govuk-link" %>
+        <% else %>
+          <ul class="govuk-summary-list__actions-list">
+            <li class="govuk-summary-list__actions-list-item">
+              <a class="govuk-link" href="#">
+                Change<span class="govuk-visually-hidden"> request for <%= allocation[:training_provider_name] %></span>
+              </a>
+            </li>
+            <li class="govuk-summary-list__actions-list-item">
+              <a class="govuk-link" data-qa="view-<%= allocation[:requested] %>-confirmation"
+                 href="<%= provider_recruitment_cycle_allocation_path(@provider.provider_code,
+                                                                      @provider.recruitment_cycle_year,
+                                                                      allocation[:training_provider_code],
+                                                                      requested: allocation[:requested])
+                 %>">
+                View&nbsp;confirmation<span class="govuk-visually-hidden"> for <%= allocation[:training_provider_name] %></span>
+              </a>
+            </li>
+          </ul>
+        <% end %>
+      </td>
+    </tr>
+  <% end %>
+  </tbody>
+</table>
+<% end %>
```

Comment:
Do we need two versions of what will basically be the same table markup?

Reply:
Yep true

Reply:
I've refactored to use a single partial